src/Pages/3DPrinting/3DPrintForm.js

```diff
@@ -9,11 +9,18 @@ import {
   Input,
   Container,
   Row,
-  Col
+  Col,
 } from 'reactstrap';
 import { submit3DPrintRequest } from '../../APIFunctions/3DPrinting';
-import Header from
-'../../Components/Header/Header.js';
+import mailImage from '../3DPrinting/gmail-icon.png';
+import discordImage from '../3DPrinting/discord-icon.png';
+
+export const copyrightMessage = `Created with ❤️ by the SCE Development 
+Team | © 2021 Software and Computer Engineering Society at SJSU`;
```

Comment:
we dont have a copyright lol

to make this follow the current year no matter what, do

```suggestion
Team | Software and Computer Engineering Society at SJSU
 ${new Date().getFullYear()}`;
```